### PR TITLE
[15.0][MIG] rma_sale_analytic

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**

### CODE REVIEW

File: rma_sale_analytic/README.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Contributors
+------------
+
+* Aaron Henriquez <ahenriquez@eficent.com>
+
+
+Maintainer
+----------
+
+This module is maintained by Eficent.
```
Comment (AaronHForgeFlow) [inline rma_sale_analytic/README.rst]:
```suggestion
This module is maintained by ForgeFlow.
```

File: rma_sale_analytic/README.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+=====
+
+* Add the analytic information in the rma line or let the system fill it
+  from origin
+
+
+Contributors
+------------
+
+* Aaron Henriquez <ahenriquez@eficent.com>
```
Comment (AaronHForgeFlow) [inline rma_sale_analytic/README.rst]:
```suggestion
* Aaron Henriquez <aaron.henriquez@forgeflow.com>
```

File: rma_sale_analytic/wizards/rma_add_sale.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+from odoo import api, models
+
+
+class RmaAddSale(models.TransientModel):
+    _inherit = "rma_add_sale"
+
+    @api.model
+    def _prepare_rma_line_from_sale_order_line(self, line):
+        data = super(RmaAddSale, self)._prepare_rma_line_from_sale_order_line(line)
+        data.update(analytic_account_id=line.analytic_account_id.id)
```
Comment (AaronHForgeFlow) [inline rma_sale_analytic/wizards/rma_add_sale.py:13]: Functionally reviewed. It does not work. It does not work in v10. I think the issue in in this line, the analytic account is at order level only. Also this called is never been called. I think this other code must be called instead: https://github.com/ForgeFlow/stock-rma/blob/15.0/rma_sale/models/rma_order_line.py#L168

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Not working

File: rma_sale_analytic/wizards/rma_add_sale.py (hunk 3 above)
Comment (JasminSForgeFlow) [reply rma_sale_analytic/wizards/rma_add_sale.py:13]: it seems analytic_account_id is not available at the sale order line level.

and with the sale order, it seems working fine

image

Comment (AaronHForgeFlow) [reply rma_sale_analytic/wizards/rma_add_sale.py:13]: ok, pending to check the other flow. Creating an RMA using this
:
image

Comment (JasminSForgeFlow) [reply rma_sale_analytic/wizards/rma_add_sale.py:13]: @AaronHForgeFlow its fixed.

Thanks

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**